.github/workflows/swagger.yml

```diff
@@ -0,0 +1,34 @@
+name: Open API Specification and Documentation
+on: [push, pull_request]
+jobs:
+  swaggerci:
+    name: swagger-build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup go-swagger
+      -  uses: minchao/setup-go-swagger@v1
+        with:
+          version: v0.26.1
+
+  build:
+    name: Swagger-ci
+    runs-on: ubuntu-latest
+    needs: [swaggerci]
+    steps:
+    - name: swagger-build
+      uses: actions/checkout@v2
+      run: |
+          sudo apt install snapd
+          sudo snap install go-swagger
+          sudo apt update
+      uses: actions/cache@v1
+      with:
+        path: |
+          ./handlers
+          ./models
+        run: swagger generate spec -o ./helpers/swagger.yaml --scan-models

```

Comment:
Move two spaces to the left of this line and the below ones. They should be at the same level of `with`